widgetry/src/style/button_style.rs

```diff
@@ -0,0 +1,135 @@
+use geom::CornerRadii;
+
+use crate::{
```

Comment:
The diff was too much for git, but this was the old `widgetry/src/style/buttons.rs` file renamed and with the StyledButtons trait deleted.